src/apply.jl

```diff
@@ -39,22 +39,36 @@ end # loop
 # TimeArray <--> Scalar
 for op in [MATH_ALL; COMPARE_DOTONLY]
     @eval begin
-        function ($op){T<:Number,N}(ta::TimeArray{T,N}, var::Number)
+        function broadcast(::typeof($op), ta::TimeArray{T, N},
+                           var::Number) where {T <: Number, N}
             cnames  = [name * string($op) * string(var) for name in ta.colnames]
-            vals = ($op)(ta.values, var)
+            vals = broadcast($op, ta.values, var)
             TimeArray(ta.timestamp, vals, cnames, ta.meta)
         end # function
+
+        # FIXME: make some function allowed dot version only
+        # e.g. `ta > 42` is not allowed, but `ta .> 42` does
+        if Symbol($op) ∉ [$MATH_DOTONLY; $COMPARE_DOTONLY]
+            ($op)(ta::TimeArray{T, N}, var::Number) where {T <: Number, N} =
```

Comment:
Try overloading `broadcast(::typeof(isless), ...)`, since `>` is defined in terms of `<`, which is just an alias for `isless`.